eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -148,6 +148,21 @@ public FakeMaven withTojoAttribute(final String attribute, final Object value) {
         return this;
     }
 
+    /**
+     * Sets foreign with path.
+     *
+     * @param path Tojo attribute.
+     * @return The same maven instance.
+     */
+    public FakeMaven withForeignPath(final Path path) {
```

Comment:
@PickBas this method is very similar to `withProgram(Path)`. Doesn't it do the job?

Reply:
Removed. Check please, @mximp